python/graphstorm/eval/utils.py

```diff
@@ -315,11 +318,10 @@ def broadcast_data(rank, world_size, data_tensor):
         dtype=data_tensor.dtype,
         device=device) for size in data_size]
     data_tensors = [data_tensor for _ in data_size]
-    if th.distributed.get_backend() == "gloo":
+    if get_backend() == "gloo":
         alltoallv_cpu(rank, world_size, gather_list, data_tensors)
-    else: #th.distributed.get_backend() == "nccl"
+    else: #get_backend() == "nccl"
         alltoallv_nccl(rank, world_size, gather_list, data_tensors)
 
```

Comment:
We don't need to define alltoallv_nccl. We can directly use Torch's alltoall for nccl backend. You can take a look at my distDGL's PR https://github.com/dmlc/dgl/pull/5929/files

Reply:
This is helpful.